### PR TITLE
Added 'censorship' to society

### PR DESCRIPTION
**Pull request by du82**
I've added #censorship to the list of public chats under Society because censorship is something that is frequently experienced and should be talked about and scrutinized.

### CODE REVIEW

File: resources/chats.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
       "politics",
       "hongkong",
       "privacy"
```
Comment (hesterbruikman) [inline resources/chats.json:46]: Looks like there's a comma missing. When testing the PR build I'm no longer seeing any public chats listed.

Screenshot_20210927-124835_Status PR

**Review by hesterbruikman — CHANGES_REQUESTED** [review]
Thanks for the PR @ducheng0 There's an issue, with the list not showing in the PR build. I think it's because of a missing comma. Can you please take a look at that?

As a heads up: we'll likely remove this list of public chats at some point because they are vulnerable to spam. Also, ironically, the current approach of a list being maintained for which you have to submit a PR that needs to be approved is, in itself not censorship resistant. The `communities` feature is the way forward 💪 It'll come with a dapp for independent community curation; For ranking, communities are fully permissionless to create as are public chats currently.

**Comment by du82** [comment]
> Thanks for the PR @ducheng0 There's an issue, with the list not showing in the PR build. I think it's because of a missing comma. Can you please take a look at that?
> 
> As a heads up: we'll likely remove this list of public chats at some point because they are vulnerable to spam. Also, ironically, the current approach of a list being maintained for which you have to submit a PR that needs to be approved is, in itself not censorship resistant. The `communities` feature is the way forward  It'll come with a dapp for independent community curation; For ranking, communities are fully permissionless to create as are public chats currently.

I was talking with one of the devs about the new community curation feature, and I'm quite excited! Maybe offer options to rank them based on amount of users in them, how much is staked, and when they were added to the list. Making Dapps look more like WeChat's mini-programs would also be awesome (having a bottom nav bar, and fullscreen experience with options in the top right). I'll figure out how to use Figma and send over my ideas on a UI.

For some reason I can't update the pull request, so you can find the updated code here: https://github.com/ducheng0/status-react/commit/1606c0711ef344bbca62c0059153c7cbf5d699f2

Thanks

**Comment by ducheng0** [comment]
I'm going to close this pull request because I've never used Git to collaborate with others on code, and I don't know what I'm doing or how this works. Help would be much appreciated :)

**Pull request closed without merging**